msteams-platform/sbs-inmeeting-document-signing.yml

```diff
@@ -0,0 +1,473 @@
+    * Purchase agreements.
+    * Purchase orders.
+    
+    The template is also applicable for other scenarios, such as for reviewing and approving code samples and collaborative documents.
+    The users can customize to incorporate other types of signatures: 
+    * Handwritten signatures.
+    * Certificate-based signatures.
+
+    This step-by-step guide helps you to build in-meeting document signing app. You'll see the following output after you've completed this guide:
+
+      :::image type="content" source="~/assets/images/sbs-inmeeting-doc-signing/final-output.png" alt-text="Final Output":::
+
+- title: Prerequisites
+  durationInMinutes: 1
+  content: |
+    Ensure you install the following tools for the development environment:  
+
+    * [Latest version of ngrok](https://ngrok.com/download) or any equivalent tunneling solution.
+
+      > [!NOTE]
+      > After downloading ngrok, sign up and install [authtoken](https://ngrok.com/download).
+
+    * [Latest version of Visual Studio](https://visualstudio.microsoft.com/downloads/).
+    * [Latest version of.NET Core SDK](https://dotnet.microsoft.com/download).
+
+    Ensure you have created the following accounts to set up your development environment: 
+
+    * [Microsoft Teams](https://teams.microsoft.com/) account.
+
+      > [!TIP]
+      > Ensure the Microsoft Teams account isn't a guest account.
+
+    * [Microsoft 365 developer account](https://developer.microsoft.com/en-us/microsoft-365/dev-program). If you don't have that, you need have access to a Teams account with the appropriate permissions to install an app.
+
+    ## Set up your Teams development tenant
+
+       A **tenant** is like a space or a container where you chat, share files, and run meetings for your organization in Teams. You can also sideload and test the app.
+
+    ### Check for sideloading option
+
+       After creating the app, you must load your app in Teams without distributing it. It's known as sideloading. Sign in to your Microsoft 365 account to view this option.
+
+       > [!NOTE]
+       > Sideloading is necessary for previewing and testing apps in Teams local environment. Enable Sideloading to preview and test your app in Teams locally.
+
+       Do you already have a tenant, and do you have the admin access? Let's check if you really do!
+
+    To verify sideload apps in Teams:
+
+    1. In the Teams client, select **Store**.
+    1. Select **Manage your apps**.
+    1. Select **Publish an app**.
+    1. Look for the option to **Upload a custom app**. If you see the option, sideloading apps is enabled.
+
+        :::image type="content" source="~/assets/images/bots/upload-custom-app1.png" alt-text="Illustration shows the option to upload a custom app in Teams.":::
+
+        > [!NOTE]
+        > Contact Teams administrator, if you don't find the option to upload a custom app.  
```

Comment:
It might be more correct to include the "your" so that it's clear it's an issue they need to bring up with their admin not some Microsoft Teams admin. E.g "Contact _your_ Teams administrator,..."